packages/react-composites/tests/browser/common/utils.ts

```diff
@@ -368,6 +370,9 @@ export async function stableScreenshot(
   if (stubOptions?.hideVideoLoadingSpinner !== false) {
     await hideVideoLoadingSpinner(page);
   }
+  if (stubOptions?.awaitFileTypeIcon !== false) {
```

Comment:
avoid the `!== false` here and just have `if (stubOptions?.awaitFileTypeIcon)`, the reason the hideVideoLoadingSpinner has it is because by default all tests should hide it. Whereas here we want to be "opt in" and only await if a test explicitly wants to